src/performance/PerformanceContextMap.ts

```diff
@@ -0,0 +1,6 @@
+type PerformanceContextKeys = 'initialRoute';
+/**
+ * Map object for saving additional performance data along the way
+ * for use in performance tracking
+ */
+export const PerformanceContextMap = new Map<PerformanceContextKeys, string>();
```

Comment:
So for now we only allow one key - `initialRoute` - with string value. Right?

And the goal of this is to have a singleton key value store for values that we can access anywhere and pass along to performance measurements?

I am wondering what are potential risks here, for example exploding memory usage over time of running the app, but I guess no risk for now?

Reply:
Yes, that's the exact use case I see. I didn't want to pass initialRoute to splashscreen hook.

Reply:
On memory side it won't explode since it will only hold a finite number of keys mostly just a couple